tfhe/docs/fundamentals/serialization.md

```diff
@@ -213,3 +213,92 @@ fn server_function(serialized_data: &[u8]) -> Result<Vec<u8>, Box<dyn std::error
     Ok(serialized_result)
 }
 ```
+
+## Safe serialization/deserialization
+
```

Comment:
is this new ? 🤔

Reply:
Yes, introduced in f244102cbc

Reply:
why do we have the diff here ?

Reply:
Oh an issue with the rebase probably sorry